src/components/TodosContext/TodosContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo, useState } from 'react';
+import { todos as todosFromServer } from '../../api/Todos';
+import { Todo } from '../../types/Todo';
+
+interface ITodosContext {
+  todosStatus: string,
+  todos: Todo[],
+  setTodosStatus: React.Dispatch<React.SetStateAction<string>>,
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>,
+}
+
+export const TodosContext = React.createContext<ITodosContext>({
```

Comment:
let's create the custom hook it will help us to avoid redundant import
```
export const useTodos = (): ITodosContext => React.useContext(TodosContext)
```